File: Heat/Models/Document.vb
Namespace Models

    Public MustInherit Class Document
        Property ID As Integer

        Property TemporaryNumber As Integer
        Property TemporaryStringNumber As String
        Property Number As Integer
        Property StringNumber As String
        Property Customer As Customer
        Property PrintDate As DateTime
        Property Numerator As Numbering
        Property CreateUser As HeatUser
        Property CreationDate As DateTime
        Property LastModifyUser As HeatUser
        Property LastModifyDate As DateTime
        Property State As DocumentState
    End Class

    Public Enum DocumentState
        Inserted = 0
        Confirmed = 1
        Deleted = 3

    End Enum
End Namespace

File: Heat.ConvertedToC#/Models/Document.cs
using System;
namespace Heat.Models
{

    public abstract class Document
	{
		public int ID { get; set; }

		public int TemporaryNumber { get; set; }
		public string TemporaryStringNumber { get; set; }
		public int Number { get; set; }
		public string StringNumber { get; set; }
		public Customer Customer { get; set; }
		public DateTime PrintDate { get; set; }
		public Numbering Numerator { get; set; }
		public HeatUser CreateUser { get; set; }
		public DateTime CreationDate { get; set; }
		public HeatUser LastModifyUser { get; set; }
		public DateTime LastModifyDate { get; set; }
		public DocumentState State { get; set; }
	}

	public enum DocumentState
	{
		Inserted = 0,
		Confirmed = 1,
		Deleted = 3

	}
}

